jpa/hibernate7/src/main/java/org/wildfly/persistence/jipijapa/hibernate7/VirtualFileSystemArchiveDescriptor.java

```diff
@@ -82,4 +82,43 @@ public InputStreamAccess getStreamAccess() {
             archiveContext.obtainArchiveEntryHandler( entry ).handleEntry( entry, archiveContext );
         }
     }
+
+    /**
+     * Example call from org.hibernate.boot.archive.scan.internal.DisabledScanner that would be used to look up orm.xml
+     * in persistence unit root (same path as persistence.xml).
+     *
+     * Example usage:
+     *     final ArchiveEntry entry = archiveDescriptor.findEntry( "META-INF/orm.xml" );
+     * @param path is expected to be META-INF/orm.xml
+     * @return
```

Comment:
Since the PR seems to require changes anyway, you might as well want to fix this missing javadoc issue.

Reply:
The org.hibernate.boot.archive.spi.ArchiveDescriptor has no javadoc for this method but I could change the comment to not look like a javadoc comment.  It is more of a note to myself or others in case other files are looked up in the future.